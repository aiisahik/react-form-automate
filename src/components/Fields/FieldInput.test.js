import React from 'react';
import renderer from 'react-test-renderer';
import { FieldInput } from '.';

test('Field renders standard input', () => {
    const fieldData = {
        type: 'input',
        fieldName: 'username',
        label: 'Email',
        validations: {}
    };
    const component = renderer.create(
        <FieldInput
            {...fieldData}
        />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});