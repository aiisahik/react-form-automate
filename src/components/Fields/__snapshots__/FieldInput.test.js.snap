// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Field renders standard input 1`] = `
<div
  className="form-group"
>
  <label
    className=" form-control-label"
    htmlFor="username"
  >
    Email
  </label>
  <input
    className="form-control"
    defaultValue={undefined}
    maxLength="60"
    name="username"
    onBlur={undefined}
    onChange={undefined}
    onKeyDown={undefined}
    placeholder={undefined}
    required={undefined}
    type="input"
  />
  <span />
</div>
`;
